presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -154,6 +155,11 @@ public SystemSessionProperties(
                             return concurrency;
                         },
                         value -> value),
+                integerSessionProperty(
+                        LOCAL_EXCHANGE_CONCURRENCY,
+                        "Number of local parallel hash partitioners per local exchange",
+                        1,
```

Comment:
Is 1 a good default value in general? Is there any reason why it can't be higher than that?
Since default task.concurrency is 16 now, do we need more local concurrency as well?